addons/web/static/tests/views/fields/many2one_field_tests.js

```diff
@@ -713,6 +713,38 @@ QUnit.module("Fields", (hooks) => {
         }
     );
 
+    QUnit.test(
+        "many2ones in list views: create a new record with a context",
+        async function (assert) {
+            await makeView({
+                type: "list",
+                resModel: "partner",
+                serverData,
+                arch: `
```

Comment:
I would add a context key here (i.e. a context props for the view), so check that the field context is correctly combined with the view context.